src/test/java/AuthenticationUtilityTest.java

```diff
@@ -0,0 +1,117 @@
+import com.google.sps.utility.AuthenticationUtility;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+/**
+ * Class to Test the Authentication Utility functions TODO: Find a way to mock the Google
+ * verification service to create unit tests for AuthenticationUtility.verifyUserToken()
+ */
+@RunWith(JUnit4.class)
+public final class AuthenticationUtilityTest {
+
+  // Will mock an HttpServletRequest to be passed to utilities
+  private final HttpServletRequest request = Mockito.mock(HttpServletRequest.class);
+
+  private final Cookie[] correctCookies =
```

Comment:
Nit: Prefer lists to arrays (and below) (https://www.informit.com/articles/article.aspx?p=2861454&seqNum=3)

Reply:
These are used as the returns of a mock object, where the mocked method returns a Cookie[] array, not a list.

I could declare these as a list, and then cast them to an array as the return from the mock. Is this still preferred?